### ISSUE TITLE
a11y.js missing function declaration

### ISSUE THREAD

**Issue by leahyjulian** [adaptlearning/adapt_framework#2573]
### Subject of the issue/enhancement/features
Course wont load in IE11 due to missing function declaration
Current:  Line 743 a11y.js 
```sh
setPopupCloseTo($focusElement) {
    return this._popup.setCloseTo($focusElement);
}
```

Should be:
```sh
setPopupCloseTo: function($focusElement) {
    return this._popup.setCloseTo($focusElement);
}
```

### Your environment
* version (4.4.0/Framework)
* IE11
* Windows

### Steps to reproduce
Launch in IE11

### Expected behaviour
View course in IE11

### Actual behaviour
Does not load course

**Comment by chetan-hajare** [comment]
The issue related to this has been raised - https://github.com/adaptlearning/adapt_framework/issues/2570
and the PR for same is here - https://github.com/adaptlearning/adapt_framework/pull/2571

**Comment by oliverfoster** [comment]
duplicate of #2570
